src/client/services/feature/feature.js

```diff
@@ -61,7 +62,7 @@ var FeatureService = SparkBase.extend(
     return this.request({
       method: 'POST',
       api: 'feature',
-      resource: 'features/users/' + this.spark.device.userId + '/developer',
```

Comment:
typically, in order to prove my fix works, i undo the fix, and poke the tests until they fail in the way i'm trying to fix, then put the fix back